src/main/java/org/opendaylight/yangtools/triemap/Constants.java

```diff
@@ -51,7 +46,7 @@ private Constants() {
     static final int MAX_DEPTH = 7;
 
     static {
-        verify(LEVEL_BITS == IntMath.log2(BITMAP_BITS, RoundingMode.UNNECESSARY));
-        verify(MAX_DEPTH == IntMath.divide(HASH_BITS, LEVEL_BITS, RoundingMode.CEILING));
+        if (LEVEL_BITS != (int) (Math.log(BITMAP_BITS)/Math.log(2))){ throw new IllegalStateException(); }
```

Comment:
Same here: create a method. Do not ever write ifs in this way -- it is utterly unreadable.